Source/Core/DolphinQt/Config/Graphics/AdvancedWidget.cpp

```diff
@@ -338,11 +341,18 @@ void AdvancedWidget::AddDescriptions()
   static const char TR_LOAD_GRAPHICS_MODS_DESCRIPTION[] =
       QT_TR_NOOP("Loads graphics mods from User/Load/GraphicsMods/.<br><br><dolphin_emphasis>If "
                  "unsure, leave this unchecked.</dolphin_emphasis>");
-  static const char TR_INTERNAL_RESOLUTION_FRAME_DUMPING_DESCRIPTION[] = QT_TR_NOOP(
-      "Creates frame dumps and screenshots at the internal resolution of the renderer, rather than "
-      "the size of the window it is displayed within.<br><br>If the aspect ratio is widescreen, "
-      "the output image will be scaled horizontally to preserve the vertical resolution.<br><br>"
-      "<dolphin_emphasis>If unsure, leave this unchecked.</dolphin_emphasis>");
+  static const char TR_FRAME_DUMPS_RESOLUTION_TYPE_DESCRIPTION[] = QT_TR_NOOP(
+      "Selects how frame dumps (videos) and screenshots are going to be captured.<br>"
+      "If the video resolution during a recording, multiple videos files will be created."
+      "<br><br><b>Window Resolution</b>: Uses the output window resolution (without black bars)."
+      "<br><b>Aspect Ratio Corrected Internal Resolution</b>: Uses the Internal Resolution (XFB "
+      "size), and corrects it by the target aspect ratio."
+      "<br><b>Raw Internal Resolution</b>: Uses the Internal Resolution (XFB size)"
+      " without corretting it with the target aspect ratio, "
+      "thus dumps will appear more crips.<br>This is "
+      "mostly meant  to be used in case you need your videos to be in the highest quality possible,"
```

Comment:
"mostly meant  to be used" double space.

Reply:
thx. fixed all.